src/pages/documentation/mobile-core/signal/api-reference.md

```diff
@@ -27,19 +27,18 @@ Flutter
 <Tabs query="platform=flutter&api=extension-version"/> --->
 
 ## registerExtension
+<InlineAlert variant="warning" slots="text"/>
+
+This API is only available in Android and is deprecated in the latest version of the Signal extension. Use [`MobileCore.registerExtensions()`](../api-reference.md#registerextensions) instead.
```

Comment:
This is more of a question for me, rather than a comment about this - but how come it's not deprecated for Android as well? Shouldn't there be parity between the iOS and Android APIs?

Reply:
😅  My bad. I can now see how this may be confusing! The intention was to convey that this is an Android only api which is deprecated (i.e exists as part of the 2.x libraries but is deprecated & not recommended). 
I am open to rewording suggestions here ...